### ISSUE TITLE
[Discussion] How does spleeter handle stereo files when training?

### ISSUE THREAD

**Issue by dustyny** [deezer/spleeter#756]
I'm creating a dataset for training my own model. At the moment my dataset is comprised of mono loops. I'm trying to figure out if I should be generating stereo but i don't know how Spleeter handles mono vs stereo. I've tried digging through the code but I'm a novice with Tensorflow and this is a complex project. 

Does it train on a stereo wav or does it split the wav up into 2 mono wavs (Left & Right channels)? 
If it uses both channels at once are they in context of one another? If I have sounds panned hard left & right does that matter? 

If I have 1,000 stereo sets does it treat it as 2,000 examples or just 1,000? 

Much appreciated..

**Comment by dustyny** [comment]
@romi1502 thank you for that answer, very helpful!. I have started training, my test run on 5k mono drum loops ended with these results.. TBH I don't know what they mean, what does good look like? 

absolute_difference = 1.7624866, 
global_step = 200000, 
loss = 1.7624866, 
brass_perc_spectrogram = 0.3552358, 
kicks_spectrogram = 0.3987537, 
pitched_perc_spectrogram = 0.5570321, s
nares_claps_spectrogram = 0.45146513

If the data is only being trained on one channel do you think that could improve or reduce accuracy?  I'm training on 55k mono loops now. Hopefully that will give a good result 🤞.
